underlay/src/main/resources/config/datamapping/sd/entity/bmi/entity.json

```diff
@@ -7,7 +7,7 @@
     { "name": "GRID", "dataType": "STRING", "valueFieldName": "person_source_value"},
     { "name": "is_clean", "dataType": "INT64", "displayFieldName": "is_clean_name", "isComputeDisplayHint": true },
     { "name": "date", "dataType": "TIMESTAMP", "valueFieldName": "measurement_date" },
-    { "name": "value_numeric", "dataType": "DOUBLE", "valueFieldName": "value_as_number", "isComputeDisplayHint": true },
+    { "name": "value_numeric", "dataType": "DOUBLE", "valueFieldName": "value_as_number", "isComputeDisplayHint": true, "displayHintRangeMin": 5, "displayHintRangeMax": 95 },
```

Comment:
Was this a request from the SD team?

Reply:
No, but this was super wide because it has some huge outliers and didn't fit in the new narrow setup.